application-services/fetch-application-services.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -euo pipefail
+
+TASKCLUSTER_HOST="https://firefox-ci-tc.services.mozilla.com"
+
+function download_megazord {
+  local PROJECT="${1}"
+  local URL="${TASKCLUSTER_HOST}/api/index/v1/task/project.application-services.v2.${PROJECT}.${APPLICATION_SERVICES_BUILD_ID}/artifacts/public%2Fbuild%2F${PROJECT}.zip"
+
+  echo download_megazord:
+  echo PROJECT "${PROJECT}"
+  echo URL "${URL}"
+
+  curl --proto '=https' \
+      --tlsv1.2 \
+      -sSL \
+      --output "${PROJECT}.zip" \
+      "${URL}"
+
+  unzip "${PROJECT}.zip" -d "/tmp/${PROJECT}"
+}
+
+function prepare_megazord {
+  local PROJECT="${1}"
+  local TARGET="${2}"
+  local LIB_DIR="/tmp/${PROJECT}/${TARGET}"
+
+  if [[ ! -d "${LIB_DIR}" ]]; then
+    echo >&2 "Could not find libraries for target ${TARGET} at ${LIB_DIR}"
+    exit 1
+  fi
+
+  mkdir -p "${PROJECT}/lib"
+  cp /tmp/"${PROJECT}"/*.py "${PROJECT}"
+  cp /tmp/"${PROJECT}"/"${TARGET}"/* "${PROJECT}/lib"
```

Comment:
I think the `.so` file needs to be in the same directory as the python files.

Also, consider moving the additional python files (in `src` and `lib`, if present) at this point.

Suggest 

```suggestion
local SO_FILE=$(/tmp/$PROJECT/scripts/detect-target.sh $PROJECT)
cp "/tmp/$PROJECT/$TARGET/$SO_FILE" "$PROJECT"
```